source/agora/node/RemoteNode.d

```diff
@@ -148,4 +148,28 @@ class RemoteNode
             return false;
         }
     }
+    
+    /***************************************************************************
+
+        Send a message
+
+        Returns:
```

Comment:
This function does not return anything, so you can remove the `Returns` comment in here.